app/data/jsonSchemaForm/projectMilestoneUiSchema.ts

```diff
@@ -10,10 +10,16 @@ const projectMilestoneUiSchema = {
     "certifierProfessionalDesignation",
     "submittedDate",
     "totalEligibleExpenses",
+    // "totalEligibleExpensesToDate",
+    // "totalGrossPaymentAmountToDate",
+    // "totalNetPaymentAmountToDate",
```

Comment:
Leftover?!

Reply:
Nope they'll need to be added later.  Just added them for ordering and sanity.